lib/orator/index.js

```diff
@@ -414,6 +420,14 @@ class Orator {
       `Hello! The prefix is \`${prefix}\`, try \`${prefix}help\``
     )
   }
+
+  async _missingWebhookPerms (response, bot, msg) {
```

Comment:
This is a bad method name. It should probably be renamed to something like `_warnMissingWebhookPerms`. Although longer, it's a clearer name

Reply:
function for one time use seems overkill